mod-source-record-manager-server/src/main/java/org/folio/services/util/EventHandlingUtil.java

```diff
@@ -116,8 +125,7 @@ public static String constructModuleName() {
   }
 
   public static KafkaProducer<String, String> createProducer(String eventType, KafkaConfig kafkaConfig) {
-    String producerName = eventType + "_Producer";
-    return KafkaProducer.createShared(Vertx.currentContext().owner(), producerName, kafkaConfig.getProducerProps());
+    return new SimpleKafkaProducerManager(Vertx.currentContext().owner(), kafkaConfig).createShared(eventType);
```

Comment:
I think SimpleKafkaProducerManager is supposed to be a singleton.

Reply:
Correct if am wrong, but is it good idea to do singleton in util class?

Reply:
I see your point. since a new kafka config is one of the inputs to this method.